packages/hyperion-autologging/src/ALSurface.ts

```diff
@@ -282,13 +283,23 @@ export function init(options: InitOptions): ALSurfaceHOC {
       }, [domAttributeName, domAttributeValue]);
     }
 
+    ReactModule.useLayoutEffect(() => {
+      const nodeRefCurrent = props.nodeRef?.current;
+
+      if (nodeRefCurrent == null) {
+        return;
+      }
+      nodeRefCurrent.setAttribute(domAttributeName, domAttributeValue);
+    }, [props.nodeRef]);
```

Comment:
merging with the above code also has the benefit of proper dependency! You need also the `domAttribute{Name|Value}` here in the list.